_slides/09-seshakus.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+I am Sesha Kintali
 
```

Comment:
This is kaushik, changed your code abd add comment

Reply:
Good